atomics/T1087/T1087.yaml

```diff
@@ -71,11 +81,14 @@ atomic_tests:
     output_file:
       description: Path where captured results will be placed
       type: Path
-      default: ~/loot.txt
+      default: /tmp/T1087.txt
   executor:
     name: sh
     command: |
-      lastlog > #{output_file}
+      if [ -x "$(command -v lastlog)" ]; then lastlog > #{output_file}; fi;
+      cat #{output_file} 2>/dev/null
+    cleanup_command: |
+      rm -f #{output_file} 2>/dev/null
```

Comment:
Could we add the check for binary existing as actual prereqs as you do in some of the later files in this commit?

Reply:
point taken